nservicebus/recoverability/index.md

```diff
@@ -0,0 +1,112 @@
+
+NOTE: When a message cannot be deserialized, it will bypass all retry mechanisms and the message will be moved directly to the error queue.
+
+## Immediate Retries
+
+NServiceBus automatically and immediately retries the message when an exception is thrown during message processing for up to five times by default. This value can be configured through `app.config` or via code. For more information about how to configure immediate retries, refer to [configure immediate retries](/nservicebus/recoverability/configure-immediate-retries.md).
+
+Note: The configured value describes the minimum number of times a message will be retried. Especially in environments with competing consumers on the same queue, there is an increased chance of retrying a failing message more times across the endpoints.
+
+### Transport transaction requirements
+
+The Immediate Retry mechanism is implemented by making the message available for consumption again at the top of the queue, so that the endpoint can process it again immediately. Immediate Retries cannot be used when transport transactions are disabled. For more information about transport transactions, refer to [transport transaction](/nservicebus/transports/transactions.md).
+
+## Delayed Retries
+
+Delayed Retries introduces another level of retry mechanism for messages that fail processing. Delayed Retries picks up the message and defers its delivery, by default first for 10 seconds, then 20, and lastly for 30 seconds, then returns it to the original worker queue.
+
+For example, if there is a call to an web service in the handler, but the service goes down for five seconds just at that time. Without Delayed Retries, the message is retried instantly and sent to the error queue. With Delayed Retries, the message is instantly retried, deferred for 10 seconds, and then retried again. This way, when the Web Service is available the message is processed just fine.
+
+For more information about how to configure delayed retries, refer to [configure delayed retries](/nservicebus/recoverability/configure-delayed-retries.md).
+
+NOTE: Retrying messages for extended periods of time would hide failures from operators, thus preventing them from taking manual action to honor their Service Level Agreements. To avoid this, NServiceBus will make sure that no message is retried for more than 24 hours before being sent the error queue.
```

Comment:
https://github.com/Particular/NServiceBus/blob/release-6.0.0/src/NServiceBus.Core/Recoverability/DefaultRecoverabilityPolicy.cs#L83

yes and we even have tests for that